### PR TITLE
Update tls cert secret validation due VerifyCertSecrets() change

### PR DESCRIPTION
**Pull request by stuggi**
Depends-On: https://github.com/openstack-k8s-operators/lib-common/pull/559

Jira: [OSPRH-9991](https://issues.redhat.com//browse/OSPRH-9991)

### CODE REVIEW

File: pkg/openstack/common.go [hunk 1]
```diff
@@ -248,25 +248,25 @@ func EnsureEndpointConfig(
...
+					_, err := validateSecret.ValidateCertSecret(ctx, helper, instance.GetNamespace())
 					if err != nil {
+						if k8s_errors.IsNotFound(err) {
+							instance.Status.Conditions.Set(condition.FalseCondition(
+								corev1.OpenStackControlPlaneCustomTLSReadyCondition,
+								condition.RequestedReason,
+								condition.SeverityInfo,
+								corev1.OpenStackControlPlaneCustomTLSReadyWaitingMessage,
+								ingressOverride.TLS.SecretName))
+							return endpoints, ctrl.Result{RequeueAfter: time.Duration(10)}, nil
```
Comment (abays) [inline pkg/openstack/common.go:260]: Does this duration need `* time.Second` as well?

File: pkg/openstack/common.go [hunk 2]
```diff
@@ -284,11 +284,12 @@ func EnsureEndpointConfig(
...
 				if !ed.Route.Create && (tlsConfig.API.Public.SecretName != nil && *tlsConfig.API.Public.SecretName != "") {
 					ed.Service.TLS.SecretName = tlsConfig.API.Public.SecretName
-					_, ctrlResult, err := ed.Service.TLS.GenericService.ValidateCertSecret(ctx, helper, instance.GetNamespace())
+					_, err := ed.Service.TLS.GenericService.ValidateCertSecret(ctx, helper, instance.GetNamespace())
 					if err != nil {
-						return endpoints, ctrlResult, err
-					} else if (ctrlResult != ctrl.Result{}) {
-						return endpoints, ctrlResult, nil
+						if k8s_errors.IsNotFound(err) {
+							return endpoints, ctrl.Result{RequeueAfter: time.Duration(10)}, nil
```
Comment (abays) [inline pkg/openstack/common.go:290]: Same question here about time duration unit

File: pkg/openstack/common.go (hunk 1 above)
Comment (stuggi) [reply pkg/openstack/common.go:260]: oh yes, thanks

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**